app/views/home/companies.md

```diff
@@ -0,0 +1,56 @@
+> Audience: Employees of a company that use and depend on open source software (a.k.a. every company that employs developers). Specifically, this page should be a link that people can send their bosses/coworkers/employees to help them learn what open source is, why it's so important to their company, and why their employees should take time to participate in Open Source Friday.
+
+> What are you asking me to do?
+
+Take one day this week to invest in the open source software you use and the community of people that make it great.
+
+> Why should I care?
+
+## Your business depends on it
```

Comment:
You didn't ask but:

IMO business hits home that this impacts on the bottom line, and if this page is for 'decision makers' rather than engineer/developer-types then that might resonate more with them. If it were me (and perhaps in part it is) I would prioritise engagement over inclusivity. Is it true that non-profits, universities etc. are less adverse to committing time to open source? Perhaps.